app/cosmos/index.js

```diff
@@ -5,8 +5,6 @@ const { filesDatabase } = require('./databases/files')
 const cosmosClient = async () => {
   try {
     const cosmos = {}
-    // applications, payments, and preferences databases are not not currently used due to
-    // the lack of current RU's for cosmos in SND4. They can be renabled when required using the same pattern.
```

Comment:
Removed this comment and mentioned it in the README so it's there for future reference.